CHANGELOG

```diff
@@ -11,6 +11,7 @@ about future releases, check `milestones`_ and :doc:`/about/vision`.
 - Bug #41 - demo/accounts.py cannot be completed automatically: user
   interaction is required to activate account. Added an interactive prompt
   in demo. Removed demo/accounts.py from tests.
+- Feater #34 - add timeout option
```

Comment:
Typo: `s/Feater/Feature`.

"add timeout option" sounds very short to me. As a pydocusign user, in order to understand which changes are introduced, I'd appreciate additional details.
Perhaps something like `Feature #34 - DocuSignClient accepts a ``timeout`` option (defaults to 30 seconds). This timeout is used when performing HTTP requests to DocuSign API.`